portfolio/src/main/webapp/style.css

```diff
@@ -1,9 +1,67 @@
+html, body {
+  height: 100%;
+  background-color: #F5F5F5;
+  font-family: helvetica, sans-serif
+}
+
 #content {
   margin-left: auto;
   margin-right: auto;
-  width: 650px;
+  height: 100%;
 }
 
-#greeting-container {
+#fact-container {
   margin-top: 20px;
 }
+
+.container{
```

Comment:
Indent by 2 spaces at a time here and below: https://google.github.io/styleguide/htmlcssguide.html#Indentation